app/views/users/show.html.haml

```diff
@@ -1,34 +0,0 @@
-%body
-  = render 'navbar'
-  %h2.text-center Recommended Daily Meal Plan
-  .wrapper
-    %header.header Your calories are #{@calories}
-    %aside.aside.aside-l
-      %h3 Breakfast: 
-      =link_to(@breakTitle, @breakLink)
-      
-      %h5 Calories: #{@breakCals}
```

Comment:
Why was this file deleted?

Reply:
I converted the haml files to erb in order to make bootstrap integration easier!